docs/quickstart.md

```diff
@@ -0,0 +1,407 @@
+        - code
+        - message
+      properties:
+        code:
+          type: integer
+          format: int32
+        message:
+          type: string
+```
+
+### stats
+
+With this command, you can get a quick statistics about the structure of one or more API definition files.
+
+In your terminal, type the following:
+
+```
+$ openapi stats petstore.yaml
+```
+
+You should get the following response:
+
+```
+$ openapi stats petstore.yaml 
+Document: petstore.yaml stats:
+
+🚗 References: 3 
+📦 External Documents: 0 
+📈 Schemas: 3 
+👉 Parameters: 2 
+🔗 Links: 0 
+➡️ Path Items: 2 
+👷 Operations: 3 
+🔖 Tags: 1 
+
+petstore.yaml: stats processed in 10ms
+```
+
+Okay, now you know some basic information about your definition. But, is it valid? Are you sure that it doesn't contain errors and will be processed properly in your environment? No, you don't. We don't either. Let's check it.
+
+### lint
+
+With this command, you ensure that your OpenAPI definition files do not contain errors. 
+
+In your terminal, type the following:
+
+```
+$ openapi lint petstore.yaml
+```
+
+You should get the following response:
+
+```
+$ openapi lint petstore.yaml 
+No configurations were defined in extends -- using built in recommended configuration by default.
+
+validating petstore.yaml...
+[1] petstore.yaml:15:7 at #/paths/~1pets/get/operationIds
+
+Property `operationIds` is not expected here.
+
+Did you mean: operationId ?
+
+13 | get:
+14 |   summary: List all pets
+15 |   operationIds: listPets
+16 |   tags:
+17 |     - pets
+
+Error was generated by the spec rule.
+
+
+[2] petstore.yaml:103:7 at #/components/schemas/Error/require
+
+Property `require` is not expected here.
+
+Did you mean: required ?
+
+101 |     $ref: "#/components/schemas/Pet"
+102 | Error:
+103 |   require:
+104 |     - code
+105 |     - message
+
+Error was generated by the spec rule.
+
+
+[3] petstore.yaml:65:17 at #/paths/~1pets~1{petId/get/parameters/0/name
+
+Path parameter `petId` is not used in the path `/pets/{petId`.
+
+63 |   - pets
+64 | parameters:
+65 |   - name: petId
+66 |     in: path
+67 |     required: true
+
+Error was generated by the path-parameters-defined rule.
+
+
+[4] petstore.yaml:14:7 at #/paths/~1pets/get
+
+Operation object should contain `operationId` field.
+
+12 | /pets:
+13 |   get:
+14 |     summary: List all pets
+15 |     operationIds: listPets
+ … |     < 27 more lines >
+43 |               $ref: "#/components/schemas/Error"
+44 |   post:
+45 |     summary: Create a pet
+
+Warning was generated by the operation-operationId rule.
+
+
+petstore.yaml: validated in 72ms
+
+❌ Validation failed with 3 errors and 1 warning.
+run with `--generate-ignore-file` to add all problems to ignore file.
+```
+
+Yikes! There are 3 errors!
+
+Don't worry, they were made intentionally to show the power of the tool.
+
+Let's fix the errors. Please, make the following modifications:
+
+```sh line 15
+operationIds: --> operationId:
+```
+
+```sh line 58
```

Comment:
```suggestion
```bash line 58
```

Reply:
I've missed this myself before,,, `sh` is not supported: https://prismjs.com/#supported-languages